pipeline/splicing/calcVarPriors.py

```diff
@@ -1092,28 +1303,85 @@ def getPriorProbSpliceRescueNonsenseSNS(variant, boundaries, deNovoDonorInRefAcc
+                                                   STD_ACC_INTRONIC_LENGTH, donor=True)
                 regionEnd = refSpliceAccBounds[nextExonNum]["acceptorStart"]
                 CIDomainInRegion = isCIDomainInRegion(regionStart, regionEnd, boundaries, variant["Gene_Symbol"])
-                isDivisible = compareDeNovoWildTypeSplicePos(variant, STD_EXONIC_PORTION, deNovoDonorInRefAcc=deNovoDonorInRefAcc)
+                isDivisible = compareDeNovoWildTypeSplicePos(variant, STD_EXONIC_PORTION, STD_ACC_INTRONIC_LENGTH,
+                                                             deNovoDonorInRefAcc=deNovoDonorInRefAcc, donor=True)
                 # if truncated region includes a clinically important domain or causes a frameshift
-                if CIDomainInRegion == True or isDivisible == False:
+                if CIDomainInRegion == True:
+                    priorProb = 0.99
+                    spliceRescue = 0
+                    CIDomainInRegionFlag = 1
+                    inExonicPortionFlag = 0
+                    frameshiftFlag = 0
+                elif isDivisible == False:
                     priorProb = 0.99
                     spliceRescue = 0
-                    if isDivisible == False:
-                        frameshift = 1
+                    isDivisibleFlag = 1
+                    frameshiftFlag = 1
+                    inExonicPortionFlag = 0
+                    CIDomainInRegionFlag = 0
                 else:
-                    # possibility of splice rescue, flag variant for further splicing assays
-                    spliceFlag = 1
-                    spliceRescue = 1
-                    frameshift = 0
-                    priorProb = "N/A"
-                    enigmaClass = "N/A"
+                    # possibility of splice rescue, check MES scores
+                    frameshiftFlag = 0
+                    inExonicPortionFlag = 0
+                    CIDomainInRegionFlag = 0
+                    isDivisibleFlag = 0
+                    deNovoSpliceData = getMaxMaxEntScanScoreSlidingWindowSNS(variant, STD_EXONIC_PORTION, STD_DE_NOVO_LENGTH,
+                                                                             donor=True, deNovoDonorInRefAcc=deNovoDonorInRefAcc)
+                    altMES = deNovoSpliceData["altMaxEntScanScore"]
+                    refZScore = deNovoSpliceData["refZScore"]
+                    altZScore = deNovoSpliceData["altZScore"]
+                    if altZScore <= refZScore:
+                        # variant creates a weaker splice site than was there previously, no change in splicing
+                        priorProb = 0.99
+                        spliceRescue = 0
+                        lowMESFlag = 1
+                    else:
+                        # variant creates a stronger splice site than reference sequence
+                        if altMES < 6.2:
```

Comment:
would it make sense to store all these constants (e.g. 0.99, 6.2, etc) somewhere as variables to be referenced? they seem fragile or arbitrary at times